apps/wallet/src/ui/app/components/menu/content/PasswordInputDialog.tsx

```diff
@@ -63,21 +71,36 @@ export function PasswordInputDialog({
                                 name="password"
                             />
                         </FieldLabel>
+                        <div className="text-center mt-4">
+                            <Text
+                                variant="p2"
+                                color="steel-dark"
+                                weight="normal"
+                            >
+                                This is the password you currently use to lock
+                                and unlock your Sui wallet.
+                            </Text>
+                        </div>
                     </div>
                     <div className="flex flex-col flex-nowrap gap-3.75 self-stretch">
                         <Button
                             type="submit"
                             variant="primary"
                             size="tall"
-                            text="Continue"
+                            text={continueLabel}
                             loading={isSubmitting}
                             disabled={!isValid}
+                            after={finalStep ? null : <ArrowRight16 />}
                         />
                         <Link
-                            text="Cancel"
+                            text="Go Back"
                             color="heroDark"
                             onClick={() => {
-                                navigate(-1);
+                                if (typeof onBackClicked === 'function') {
```

Comment:
you could default `onBackClicked` to an empty function to avoid this check

Reply:
but in case `onBackClicked` is not empty it has to call `navigate(-1)` how would that work? Ideally it should default to `navigate(-1)` but not possible right?

Reply:
ah yea, missed that